src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -326,6 +326,13 @@ MavlinkReceiver::handle_message(mavlink_message_t *msg)
 	 *
 	 * Accept HIL GPS messages if use_hil_gps flag is true.
 	 * This allows to provide fake gps measurements to the system.
+	 *
+	 * SSRC:
+	 * in HIL mode, MAVLINK_MSG_ID_HIL_SENSOR only contains imu sensor data from
```

Comment:
yes, currently in our models, there is only IMU plugin, hence only this https://github.com/tiiuae/px4-gzsim-plugins/blob/70a46b91baaf44de4a9f3160928e7f95a62e060b/src/mavlink_interface.cpp#L370C6-L370C12 got called and the sensor msg only contains updated IMU field. However, we can still keep the code as is in case of any future expansion as well as avoiding much deviation from upstream in case of HITL.